packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -3497,34 +3625,36 @@ class _ExitWidgetSelectionButtonState
     milliseconds: 100,
   );
 
+  @override
+  State<_TooltipGestureDetector> createState() =>
+      _TooltipGestureDetectorState();
+}
+
+class _TooltipGestureDetectorState extends State<_TooltipGestureDetector> {
+  Timer? _tooltipVisibleTimer;
+  Timer? _tooltipHiddenTimer;
```

Comment:
cancel both of these timers in the dispose method and set them back to null.

Reply:
Done!